fixes/github-issue-create.js

```diff
@@ -47,31 +47,45 @@ async function createGithubIssue(fs, options, targets, dryRun = false) {
       this.Octokit
     )
 
+    let assignees = []
+    const assigneeSelectCount = 0
```

Comment:
Why defining a `const` count(er) with value `0`?

Reply:
I think this is completely redundant and should be refactored. Thanks!